### PR TITLE
CI fixes

### PR DESCRIPTION
**Pull request by Lenni009**

### CODE REVIEW

**Comment by Lenni009** [comment]
Fix #9

File: .github/workflows/publish-github-pages.yml [hunk 1]
```diff
@@ -10,23 +10,17 @@ on:
...
 jobs:
-  npm:
+  pages:
     runs-on: ubuntu-latest
 
     steps:
       - name: Checkout Repo
         uses: actions/checkout@v4
 
-      - name: Set up Node
```
Comment (Khaoz-Topsy) [inline .github/workflows/publish-github-pages.yml:24]: Is this not necessary? 😮

Comment (Lenni009) [reply .github/workflows/publish-github-pages.yml:24]: No, the runner has Node pre-installed

Comment (Khaoz-Topsy) [reply .github/workflows/publish-github-pages.yml:24]: That is cool, I had no idea 😮

**Pull request merged**